code/modules/clothing/head/helmet.dm

```diff
@@ -67,8 +67,9 @@
 				usr << "[alt_toggle_message] \the [src]"
 				usr.update_inv_head(0)
 				while(up)
-					playsound(src.loc, "[activation_sound]", 100, 0, 4)
-					sleep(15)
+					if(active_sound)

```

Comment:
I specified this to be above the while(up) for a reason, this is to prevent while() needlessly looping itself around 5ever